docs/user/custom-resources/apigateway/apigateway-v1beta2-adr.md

```diff
@@ -0,0 +1,118 @@
+| **rules.mutators**              |  **NO**   | Specifies the list of the Istio mutators.                                                                                                                                                                                                                                                                                                    |
+| **rules.accessStrategies**      |  **YES**  | Specifies the list of access strategies. Supported are `no_auth`, `extAuth` and `jwt` as [Istio](https://istio.io/latest/docs/tasks/security/authorization/authz-jwt/) access strategy.                                                                                                                                                      |
+| **rules.timeout**               |  **NO**   | Specifies the timeout, in seconds, for HTTP requests made to **spec.rules.path**. The maximum timeout is limited to 3900 seconds (65 minutes). Timeout definitions set at this level take precedence over any timeout defined at the **spec.timeout** level.                                                                                 |
+
+### Examples
+
+- Multiple hosts with external authorizer:
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  name: service-config
+spec:
+  gateway: kyma-system/kyma-gateway
+  hosts:
+    - api1.example.com
+    - api2.example.com
+  service:
+    name: httpbin
+    port: 8000
+  rules:
+    - path: /headers
+      methods: ["GET"]
+      mutators:
+        - ***
+      accessStrategy:
+        extAuth:
+          name: oauth2-proxy
+          restrictions:
+            authentications:
+              - issuer: https://example.com
+                jwksUri: https://example.com/.well-known/jwks.json            
+            authorizations:
+              - audiences: ["app1"]
+```
+
+- One host with jwt:
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  name: service-config
+spec:
+  gateway: kyma-system/kyma-gateway
+  hosts:
+    - app1.example.com
+  service:
+    name: httpbin
+    port: 8000
+  rules:
+    - path: /headers
+      methods: ["GET"]
+      mutators:
+        - ***
+      accessStrategy:
+        jwt:
+          authentications:
+            - issuer: https://example.com
+              jwksUri: https://example.com/.well-known/jwks.json            
+          authorizations:
+            - audiences: ["app1"]
+```
+
+- One host with noAuth:
+```yaml
+apiVersion: gateway.kyma-project.io/v1beta2
+kind: APIRule
+metadata:
+  name: service-config
+spec:
+  gateway: kyma-system/kyma-gateway
+  hosts:
+    - app1.example.com
+  service:
+    name: httpbin
+    port: 8000
+  rules:
+    - path: /headers
+      methods: ["GET"]
+      mutators:
+        - ***
+      accessStrategy:
+        noAuth: true
```

Comment:
Just for discussion: with a config like that, we would need to make a breaking change if we ever want to add `noAuth` specific fields. But we probably don't have anything on the horizon and want to keep noAuth as simple as possible.